src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs

```diff
@@ -40,6 +41,20 @@ public class InvitationsController : ControllerBase
             return this.FromResult(result);
         }
 
+        [Authorize(Roles = Permissions.IPO_READ)]
```

Comment:
Should this be COMMPKG_READ? Since we are just listing the IPOs. They will be "stopped" if trying to click on an IPO if they do not have IPO_READ